src/modules/notifications.ts

```diff
@@ -56,6 +60,22 @@ export class NotificationsAPI extends BaseClient {
       headers: { 'Content-Type': 'application/json' }
     })
   }
+
+  async putEmail(email: string) {
+    return this.fetch<Subscription>('/set-email', {
+      method: 'PUT',
+      body: JSON.stringify(email),
```

Comment:
[According to the API](https://github.com/decentraland/notifications-workers/blob/7ee172c73cd456fbedd9175cac57db424091a1af/inbox/src/controllers/handlers/unconfirmed-email-handlers.ts#L14):
```suggestion
      body: JSON.stringify({ email }),
```